cmd/bucket-targets.go

```diff
@@ -391,3 +394,35 @@ func generateARN(t *madmin.BucketTarget) string {
 	}
 	return arn.String()
 }
+
+// Returns parsed target config
+func parseBucketTargetConfig(cdata []byte) (*madmin.BucketTargets, error) {
+	var (
+		data []byte
+		err  error
+		t    madmin.BucketTargets
+	)
+	if len(cdata) == 0 {
+		return nil, nil
+	}
+	// Is rotating of creds requested?
+	if globalOldCred.IsValid() {
```

Comment:
Isn't `migrateTargetConfig` there to make sure that we don't need to do this?

Can you rotate while running?